src/app/views/realms/view/overview/RouteRealmsOverview.vue

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+import { computed, Ref, ref } from 'vue';
+import { useRoute, useRouter } from 'vue-router';
+import { Api } from '../../../../../_common/api/api.service';
+import AppButton from '../../../../../_common/button/AppButton.vue';
+import { FiresidePost } from '../../../../../_common/fireside/post/post-model';
+import AppLoadingFade from '../../../../../_common/loading/AppLoadingFade.vue';
+import {
+	createAppRoute,
+	defineAppRouteOptions,
+} from '../../../../../_common/route/route-component';
+import { $gettextInterpolate } from '../../../../../_common/translate/translate.service';
+import { kLineHeightComputed } from '../../../../../_styles/variables';
+import { ActivityFeedService } from '../../../../components/activity/feed/feed-service';
+import { ActivityFeedView } from '../../../../components/activity/feed/view';
+import AppFiresideAvatar from '../../../../components/fireside/avatar/AppFiresideAvatar.vue';
+import AppFiresideAvatarAdd from '../../../../components/fireside/avatar/AppFiresideAvatarAdd.vue';
+import { AppActivityFeedLazy } from '../../../../components/lazy';
+import AppPostAddButton from '../../../../components/post/add-button/AppPostAddButton.vue';
+import { routeRealmsViewFiresides } from '../firesides/firesides.route';
+import { useRealmRouteStore } from '../view.store';
+
+export default {
+	...defineAppRouteOptions({
+		resolver: ({ route }) =>
+			Promise.all([
+				Api.sendRequest('/web/realms/' + route.params.path),
```

Comment:
Do we need to request the realm again here? The parent route seems to be doing the same thing.

Reply:
i asked david about this. the issue is i want to always fetch the fresh data so i can update the sidebar rendered by the parent, but the parent doesn't bother reloading when switching between it's children. so i just had both children refetch and update a common store.

maybe we can do something like the parent deferring displaying the shell until the child route loads in and populates the store? this way we can just not fetch in the parent route.